jupyterlab_caip_optimizer/src/service/optimizer.ts

```diff
@@ -61,17 +59,19 @@ export class OptimizerService {
 
   async createStudy(study: Study, metadata: MetadataRequired): Promise<Study> {
     const body = JSON.stringify(study);
+    const ENDPOINT = `https://${metadata.region}-ml.googleapis.com/v1`;
     const response = await this._transportService.submit<Study>({
-      path: `${this.serverSettings.baseUrl}gcp/v1/projects/${metadata.projectId}/locations/${metadata.region}/studies?study_id=${study.name}`,
+      path: `${this.serverSettings.baseUrl}gcp/v1/proxy/${ENDPOINT}/projects/${metadata.projectId}/locations/${metadata.region}/studies?study_id=${study.name}`,
       method: 'POST',
       body,
     });
     return response.result;
   }
 
   async listStudy(metadata: MetadataRequired): Promise<Study[]> {
+    const ENDPOINT = `https://${metadata.region}-ml.googleapis.com/v1`;
     const response = await this._transportService.submit<Study[]>({
-      path: `${this.serverSettings.baseUrl}gcp/v1/projects/${metadata.projectId}/locations/${metadata.region}/studies`,
+      path: `${this.serverSettings.baseUrl}gcp/v1/proxy/${ENDPOINT}/projects/${metadata.projectId}/locations/${metadata.region}/studies`,
```

Comment:
Looking at the `ServerProxyTransportService` it seems that it handles attaching the proxy url ([link here](https://github.com/kpfromer/jupyter-extensions/blob/master/shared/src/service/transport.ts#L78)). I think the path should look like `${ENDPOINT}/projects/${metadata.projectId}/locations/${metadata.region}/studies`

Reply:
you're right, thanks for pointing out